deploy/quickstart.sh

```diff
@@ -46,12 +46,12 @@ spec:
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: service-assurance-operator
+  name: serviceassurance-operator-alpha-redhat-service-assurance-operators-openshift-marketplace
```

Comment:
Can't we work "Automatic" and "v0.1.0" in there somehow to get the grand slam?

I should have known that teaching people not to use the name for metadata, combined with the natural tendency to want to do that, would lead to this. 

https://i.imgflip.com/3lmr9r.jpg

Reply:
I'm only following what OLM already generates for a bunch of the Subscriptions :)

Reply:
Oh yeah, I'm not blaming you. This is the natural course of things :)